### PR TITLE
Chore: Improve admin docs

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Improve docs.

# Changes

* Changes in ADMIN.md

# Tests

Only docs changes.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

In the 3rd step:

> To open next version afterward, bump the libraries' version and add a note in [CHANGELOG](./CHANGELOG.md).

Should I do a patch bump on all libraries? Is that necessary? The changelog I changed it in the previous PR.

File: ADMIN.md [hunk 1]
```diff
@@ -76,6 +76,21 @@ Add new library to "nightly" and "publish" CI [workflows](https://github.com/dfi
 
 ## Release
 
-To release a version to npm, create a release in GitHub. This will start an action that build and publish the libraries.
+Steps to release new versions of the libraries.
+
+1. Create a PR bumping versions
+
+- Upgrade `"version"` in all needed packages.
```
Comment (peterpeterparker) [inline ADMIN.md:83]: Maybe something like following to be more accurate then:

```
- Upgrade `"version"` in the various library `packages.json`. If package depends on another package of ic-js, update the reference as well.
- Update `"version"` in the root `packages.json` of the project
- Run `npm i` from root directory
- Update Changelog
```

File: ADMIN.md [hunk 2]
```diff
@@ -76,6 +76,21 @@ Add new library to "nightly" and "publish" CI [workflows](https://github.com/dfi
...
+- Update intra-library dependencies.
+- Run `npm install` from root directory
+- Update Changelog
+
+Example PR: https://github.com/dfinity/ic-js/pull/475
+
+2. Create a new release in Github page.
```
Comment (peterpeterparker) [inline ADMIN.md:90]:
```
2. Create a new Github [release](https://github.com/dfinity/ic-js/releases).
```

File: ADMIN.md (hunk 1 above)
Comment (lmuntaner) [reply ADMIN.md]: done!

File: ADMIN.md (hunk 2 above)
Comment (lmuntaner) [reply ADMIN.md]: done!

**Comment by lmuntaner** [comment]
@peterpeterparker ready for another review

**Review by peterpeterparker — APPROVED** [review]
Thanks

**Pull request merged**